tools/depends/Makefile.include.in

```diff
@@ -32,7 +32,10 @@ LINK_ICONV=@link_iconv@
 ENABLE_GPLV3=@use_gplv3@
 HAS_LIBCRYSTAX=@has_libcrystax@
 
-BASE_URL=http://mirrors.xbmc.org/build-deps/sources
+ifneq ($(KODI_MIRROR),)
+  KODI_MIRROR=http://mirrors.kodi.tv
+endif
+BASE_URL=$(KODI_MIRROR)/build-deps/sources
```

Comment:
to fix OS, please change that block to
>BASE_URL=http://mirrors.kodi.tv/build-deps/sources
ifneq ($(KODI_MIRROR),)
  BASE_URL=$(KODI_MIRROR)/build-deps/sources
endif